src/v/kafka/data/rpc/client.h

```diff
@@ -63,15 +63,17 @@ class client {
       cluster::topic_properties,
       std::optional<int32_t> partition_count = std::nullopt);
 
+    ss::future<cluster::errc> update_topic(cluster::topic_properties_update);
+
     ss::future<> start();
     ss::future<> stop();
 
-private:
     ss::future<cluster::errc> try_create_topic(
```

Comment:
why does this need to be public?

Reply:
good q - i might have used it for something on a separate PoC branch. if that's the case it shouldn't be public here. will check.

Reply:
ah, I changed some stuff around in `transform/rpc/client.cc` so it uses this function to create topics.